Tools/Pipeline/Windows/MainView.cs

```diff
@@ -1041,11 +1041,56 @@ private void MainView_DragEnter(object sender, DragEventArgs e)
 
         private void MainView_DragDrop(object sender, DragEventArgs e)
         {
-            string filename = GetDropFile(e.Data, ".mgcb");
+            string filename = GetDropFile(e.Data, ".MGCB");
```

Comment:
Why this change to the case of the string literal for the extension?  `GetDropFile()` does a case-insensitive comparison anyway, so this change is redundant.  Lower-case file extensions have been the norm for the past 20 years, since MS-DOS was put to pasture.